src/controllers/taskCommand.ts

```diff
@@ -0,0 +1,42 @@
+import { env } from "../typeDefinitions/default.types";
+import { TasksResponseType } from "../typeDefinitions/task.types";
+import { formatTask, generateTaskResponseMessage } from "../utils/formatTask";
+import { discordTextResponse } from "../utils/discordResponse";
+import { fetchTasks } from "../utils/fetchTasks";
+import { getNickName } from "../utils/getNickName";
+import {
+  INVALID_NICKNAME_ERROR,
+  NO_TASKS_FOUND,
+  TASKS_FETCH_FAILED,
+} from "../constants/responses";
+
+export async function taskCommand(userId: string, env: env) {
+  try {
+    const nickName = await getNickName(userId, env);
+    if (nickName === null) {
+      return discordTextResponse(INVALID_NICKNAME_ERROR);
+    }
+    const status = "IN_PROGRESS";
+    const tasksData = await fetchTasks(nickName, status);
```

Comment:
```suggestion
export async function taskCommand(userId: string, env: env) {
const status = "IN_PROGRESS";
  try {
    const nickName = await getNickName(userId, env);
    if (nickName === null) {
      return discordTextResponse(INVALID_NICKNAME_ERROR);
    }
    const tasksData = await fetchTasks(nickName, status);
```

NIT - Can we do this if possible?

Reply:
The reason being  `const status = "IN_PROGRESS";` where status is being assigned some value and when we do this type we should try to put this type of code to be at the start of the function.

Reply:
Done